common/ECSResourcesCheck/src/main/java/ECSResources.java

```diff
@@ -0,0 +1,103 @@
+import com.amazonaws.auth.AWSCredentialsProvider;
+import com.amazonaws.auth.AWSStaticCredentialsProvider;
+import com.amazonaws.auth.BasicAWSCredentials;
+import com.amazonaws.services.ecs.AmazonECS;
+import com.amazonaws.services.ecs.AmazonECSClientBuilder;
+import com.amazonaws.services.ecs.model.*;
+import org.json.JSONObject;
+
+public class ECSResources {
+    public AmazonECS _ecsClient;
+    public String _cluster;
+
+    public ECSResources(String cluster, String aws_access_key_id, String aws_secret_access_key, String region) {

```

Comment:
Rename ECSResources to ECSService and update the constructors with the new name.

Reply:
done